### PR TITLE
smart-agent_systemd-services: init

### PR DESCRIPTION
**Pull request by tchernomax**
new module

### CODE REVIEW

File: modules/smart-agent_systemd-services/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# SYSTEMD-SERVICES SignalFx detectors
```
Comment (xp-1000) [inline modules/smart-agent_systemd-services/README.md:1]: may be `systemd` would be enough and shorter for the module name ?

File: modules/smart-agent_systemd-services/conf/readme.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+documentations:
+  - name: Smart Agent monitor
+    url: 'https://docs.signalfx.com/en/latest/integrations/agent/monitors/collectd-systemd.html'
+
+source_doc:
```
Comment (xp-1000) [inline modules/smart-agent_systemd-services/conf/readme.yaml:5]: I would add a configuration example of the monitor here 
you check an example here : https://github.com/claranet/terraform-signalfx-detectors/tree/master/modules/smart-agent_nginx-ingress

File: modules/smart-agent_systemd-services/detectors-systemd-services.tf [hunk 3]
```diff
@@ -0,0 +1,24 @@
+resource "signalfx_detector" "systemd_services" {
```
Comment (xp-1000) [inline modules/smart-agent_systemd-services/detectors-systemd-services.tf:1]: given that the detector is very basic I would have used the yaml generator to ease the creation and the review.

See https://github.com/claranet/terraform-signalfx-detectors/blob/master/docs/development.md#bootstrap-a-new-module for more information and https://github.com/claranet/terraform-signalfx-detectors/blob/master/modules/smart-agent_system-common/conf/01-cpu.yaml for an example

**Review by xp-1000 — CHANGES_REQUESTED** [review]
Overall seems good, just some remarks

File: modules/smart-agent_systemd-services/README.md (hunk 1 above)
Comment (tchernomax) [reply modules/smart-agent_systemd-services/README.md:1]: I disagree, today systemd manage A LOT of things.
Beside .service there is .timer, .path, .device, .mount, …

With this module we can only monitor services and I think it should be explicit in the module name.

Comment (xp-1000) [reply modules/smart-agent_systemd-services/README.md:1]: I thought maybe this module could grow up in future to embed more than services but ok no problem just a suggestion

**Review by xp-1000 — APPROVED** [review]
Good job thanks!

**Pull request merged**